terraform/modules/ecs/_variable.tf

```diff
@@ -0,0 +1,106 @@
+variable "appmesh_name" {
+  type        = string
+  description = ""
+}
+
+variable "availability_zones" {
+  description = "AZs"
```

Comment:
Any chance we can make this a bit more robust? I have a feeling that the "AZ" abbreviation is going to be interpreted as a state and lead to confusion with our jurisdictions. Same concern below at line 12.

Reply:
I've updated all descriptions, hopefully things are looking better now